NEWS.md

```diff
@@ -14,11 +28,27 @@
 - `<expr>$rolling_kurtosis()` (#1665).
 - `pl$explain_all()` to show a single optimized query plan from several input LazyFrames (#1666).
 - `<expr>$bin$reinterpret()` (#1664).
+- `<expr>$mode()` gains the `maintain_order` argument (#1662).
+- `pl$scan_parquet()` and `pl$read_parquet()` gain the `missing_columns` argument
+  to specify how to handle missing columns in the scanned files (#1662).
```

Comment:
I think that's duplicated with the bullet point about deprecation of `allow_missing_columns`